### DISCUSSION TITLE
Welcome to Executable Books Discussions!

### DISCUSSION THREAD

**Discussion by chrisjsewell** [executablebooks/meta#88] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**NOTE:** This discussions feature in GitHub is in [Beta development](https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/#discussions), so be patient with any bugs and expect to see improvements in the near future!

**Comment by Baalkikhaal** [comment]
Hi I am Sreekar. I intend to use to publish my PhD thesis as a Jupyter book. 😄

**Comment by simateimourian** [comment]
Hello everyone,
hope you are doing well

I have a question about building the Jupyter book.  I created a repository and HTML page for my book, but I do not know how I can create my chapters and add content to the page. I am new in this field, and I would be grateful if you have a tutorial except on the Jupyter website. Let me know.  Merci
